testing/tests/DevExpress.viz.core.series/barPoint.tests.js

```diff
@@ -355,99 +362,117 @@ QUnit.test('Point is out of boundaries on the left', function(assert) {
 
     point.translate();
 
-    assert.strictEqual(point.inVisibleArea, false, 'inVisibleArea');
+    assert.notOk(point.isInVisibleArea(), 'inVisibleArea');
     assert.strictEqual(point.y, 250, 'y');
-    assert.strictEqual(point.minY, 300, 'minY');
-    assert.strictEqual(point.height, 50, 'height');
+    assert.strictEqual(point.minY, 295, 'minY');
+    assert.strictEqual(point.height, 45, 'height');
     assert.strictEqual(point.x, 0, 'x');
     assert.strictEqual(point.width, 50, 'width');
 
     assert.equal(point.vx, 25, 'crosshair x Coord');
     assert.equal(point.vy, point.y, 'crosshair y Coord');
 });
 
-QUnit.test('Point is partially out of boundaries on the left and bottom', function(assert) {
+QUnit.test('Point is out of boundaries on the left, bar\'s right bound on left border', function(assert) {
```

Comment:
Suggested approach simplified search when we copy test's name on browser page and search through the code base
```suggestion
QUnit.test("Point is out of boundaries on the left, bar's right bound on left border", function(assert) {
```

Reply:
Eslint rise the error on this line and change it with single quote. I can change the name of the test then.